client/src/components/createOrder.vue

```diff
@@ -93,6 +93,11 @@ export default {
             }
         }
     },
+
+    mounted() {
+        this.selectedDateDeparture = new Date();
```

Comment:
```this.selectedDateArrival = new Date();``` should be added as well so the departure field won't be empty when the page loads.

Reply:
ok, done